pixi/src/checks/PageExperienceCheck.js

```diff
@@ -109,10 +109,11 @@ export default class PageExperienceCheck {
   }
 
   createReportData(apiResult) {
-    const fieldMetrics = apiResult.loadingExperience.origin_fallback
-      ? null
-      : apiResult.loadingExperience.metrics;
-    const audits = apiResult.lighthouseResult.audits;
+    const {loadingExperience, lighthouseResult} = apiResult || {};
+    const {origin_fallback: originFallback, metrics} = loadingExperience || {};
```

Comment:
This looks a bit weird. Seems like `origin_fallback` is never used? What's the real problem here that you're trying to fix?

Reply:
`origin_fallback` was already in use. See 012bd02e48ed1d3a109328b44ff3452062ffe858
It's still in use on line 114 ;)

The previous code (`const fieldMetrics = apiResult.loadingExperience.origin_fallback`) failed when there's no `loadingExperience` field available in the api result as described in #5040 See: https://www.googleapis.com/pagespeedonline/v5/runPagespeed?strategy=mobile&url=https://medinathoughts.com/

Reply:
I just needed to rename `origin_fallback` to `originFallback` to match the eslint config

Reply:
My bad. Got confused by the destructuring operator.